frontend/src/views/WheresMyGene/components/HeatMap/utils.ts

```diff
@@ -268,14 +297,18 @@ export function createYAxisOptions({
 
             const displayDepth = Math.min(depth, MAX_DEPTH);
 
-            const paddedName = " ".repeat(displayDepth * 8) + name;
+            const paddedName = getFixedWidth(
+              " ".repeat(displayDepth * 8) + name,
+              200
```

Comment:
How is this number determined? If it's related to `Y_AXIS_CHART_WIDTH_PX` (at the top of this file), could you make it scale with that variable?

Reply:
The 200 number was just what fit the best for the current wrapper/container width. But you're right, it would be better to have it scale to that constant in case we want to make the Y-axis wider